pbm/backup/backup.go

```diff
@@ -75,21 +77,26 @@ func (b *Backup) run(bcp pbm.BackupCmd) (err error) {
 		}
 	}()
 
+	stg, err := b.cn.GetStorage()
+	if err != nil {
+		return errors.Wrap(err, "unable to get PBM storage configuration settings")
+	}
+
 	ver, err := b.node.GetMongoVersion()
 	if err == nil {
 		meta.MongoVersion = ver.VersionString
 	}
 
-	stg, err := b.cn.GetStorage()
+	cfg, err := b.cn.GetConfig()
 	if err != nil {
-		return errors.Wrap(err, "unable to get backup store")
+		return errors.Wrap(err, "unable to get PBM config settings")
 	}
-	if stg.Type == pbm.StorageUndef {
+	if cfg.Storage.Type == pbm.StorageUndef {
```

Comment:
As I see `GetStorage` already have logic for checking if storage is valid. Probably that validation logic can be moved to `GetConfig` ligic?

Reply:
w